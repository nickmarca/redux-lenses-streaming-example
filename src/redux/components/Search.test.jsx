import React from 'react';
import { create } from 'react-test-renderer';
import Search from './Search';

describe('Search', () => {
    it('matches the snapshot', () => {
        const search = create(<Search />);

        expect(search.toJSON()).toMatchSnapshot();
    });
});
