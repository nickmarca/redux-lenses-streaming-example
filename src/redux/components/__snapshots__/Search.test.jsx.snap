// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search matches the snapshot 1`] = `
<div
  className="panel-block"
>
  <div
    className="field has-addons"
  >
    <div
      className="control"
    >
      <input
        className="input is-small"
        onChange={[Function]}
        placeholder="Find a message"
        type="text"
        value=""
      />
    </div>
    <div
      className="control"
    >
      <button
        className="button is-info is-small"
        onClick={[Function]}
      >
        Search
      </button>
    </div>
  </div>
</div>
`;
